string sign_type
geometry_msgs/Point sign_position
